### ISSUE TITLE
WCAG 1.3.1: Ensure that <th> elements and elements with role=columnheader/rowheader have data cells they describe (table)

### ISSUE THREAD

**Issue by azap026** [azap026/smetalabv3#104]
#### Issue
`Ensure that <th> elements and elements with role=columnheader/rowheader have data cells they describe` ([th-has-data-cells](https://dequeuniversity.com/rules/axe/4.10/th-has-data-cells?application=msftAI))
#### Target application
[Smetalab - Smart Engineering & Management](http://localhost:3000/app/guide/materials)
#### Element path
table
#### Snippet
`<table context="[object Object]" style="border-spacing: 0px; overflow-anchor: none; width: 100%; table-layout: fixed; border-collapse: separate;">`
#### How to fix
    Fix all of the following:
      Table data cells are missing or empty
#### Environment
Chrome version 143.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
